js/classes/entities/ui_ground/hud.js

```diff
@@ -67,6 +67,51 @@ class Hud {
         });
     }
 
+    // /**
+    //  * Switch the mouse icon to a crosshair.
```

Comment:
What's all this commented out code? Did you decide to not include the crosshair for consistency across browsers?

Reply:
This is an alternate implementation of the crosshair/pointer. It's commented out because it had buggy behavior. I left it in because the core idea still seems good. It just needs a little work.